app/views/content/cookies.html.erb

```diff
@@ -1,140 +1,138 @@
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
-    <h1 class="govuk-heading-xl">Cookies</h1>
-
-    <h2 class="govuk-heading-l" id="cookies-that-measure-website-use">Cookies that measure website use</h2>
-
-    <p class="govuk-body">
-    We use Google Analytics software to collect information about how you use ‘<%= service_name %>’. We do this to help make sure the site is meeting the needs of its users and to help us make improvements.
+    <h1 class="govuk-heading-l">Cookies</h1>
+    <p class='govuk-body'>
```

Comment:
Use double quotes for HTML attributes.

```suggestion
    <p class="govuk-body">
```

Reply:
🤔 Interesting that the erb linter does not pick up on this

Reply:
Yeah, that was my hope, somewhat dashed by reality.